api_spec.yaml

```diff
@@ -832,9 +832,6 @@ paths:
                 name:
                   type: string
```

Comment:
the ADR - if the scopes array is empty, you add openid to it, else don't modify the scopes array

Reply:
changes made.